src/test/java/org/sagebionetworks/bridge/notification/worker/BridgeNotificationWorkerProcessorTest.java

```diff
@@ -115,20 +117,44 @@ public void multipleUsers() throws Exception {
         when(mockBridgeHelper.getAllAccountSummaries(STUDY_ID)).thenReturn(ImmutableList.of(accountSummary1,
                 accountSummary2, accountSummary3).iterator());
 
-        doThrow(IOException.class).when(processor).processAccountForDate(STUDY_ID, DATE, accountSummary2);
+        doThrow(IOException.class).when(processor).processAccountForDate(STUDY_ID, DATE, "user-2");
 
         // Execute
         processor.accept(makeValidRequestNode());
 
         // Verify calls to processAccount()
-        verify(processor).processAccountForDate(STUDY_ID, DATE, accountSummary1);
-        verify(processor).processAccountForDate(STUDY_ID, DATE, accountSummary2);
-        verify(processor).processAccountForDate(STUDY_ID, DATE, accountSummary3);
+        verify(processor).processAccountForDate(STUDY_ID, DATE, "user-1");
+        verify(processor).processAccountForDate(STUDY_ID, DATE, "user-2");
+        verify(processor).processAccountForDate(STUDY_ID, DATE, "user-3");
 
         // Verify call to dynamoHelper.writeWorkerLog()
         verify(mockDynamoHelper).writeWorkerLog(TAG);
     }
 
+    @Test
+    public void userList() throws Exception {
+        // Pass in user list of 2 users.
+        ArrayNode userListNode = JSON_MAPPER.createArrayNode();
+        userListNode.add("user-A").add("user-B").add("user-C");
```

Comment:
What happens if the userList property isn't present in the JSON? Worth a test?

Reply:
multipleUsers() tests this code path, as does one of the integ tests.